logstash-core/lib/logstash/pipeline.rb

```diff
@@ -338,6 +341,8 @@ def worker_loop(batch_size, batch_delay)
       end
       flush_filters_to_batch(batch, :final => false) if signal.flush?
       if batch.filteredSize > 0
+
+        java_import org.apache.logging.log4j.ThreadContext
```

Comment:
any reason to have this `java_import` here?

Reply:
no, was my typo, the import is already on top of the file